static/app/utils.tsx

```diff
@@ -195,9 +195,13 @@ export function formatBytesBase10(bytes: number, u: number = 0) {
  * For billing-related code around attachments. please take a look at
  * formatBytesBase10
  */
-export function formatBytesBase2(bytes: number, fixPoints: number = 1): string {
+export function formatBytesBase2(bytes?: number, fixPoints: number = 1): string {
   const units = ['KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
   const thresh = 1024;
+  if (!bytes) {
+    return '--B';
```

Comment:
Do we know why this is happening though? Is there a byte field returning as none? If you check the issue you can follow the URL and repro it yourself in the span tree, let's be sure there isn't an upstream backend error before null guarding in the frontend

Reply:
Yup, makes sense, i'll go through it later today!